tools/earth/landcover_subindic.xml

```diff
@@ -0,0 +1,133 @@
+<tool id="landcover_subindicator" name="Land cover degradation" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="20.01" license="MIT">
+    <description>subindicator to track land changes</description>
+    <macros>
+        <token name="@TOOL_VERSION@">0.1.0</token>
+        <token name="@VERSION_SUFFIX@">0</token>
+    </macros>
+    <requirements>
+        <requirement type="package" version="1.26.3">numpy</requirement>
+        <requirement type="package" version="3.8.4">matplotlib</requirement>
+        <requirement type="package" version="1.3.10">rasterio</requirement>
+        <requirement type="package" version="2.1.14">trends_earth_algorithms</requirement>
+        <requirement type="package" version="2.1.14">trends_earth_schemas</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+            #if '$method.json_file' == "no"
+                python '$__tool_directory__/land_cover.py'
+                --raster_1 '$raster_1'
+                --raster_2 '$raster_2'
+                --json '$json'
+            #else        
+                #set $json_url = "https://raw.githubusercontent.com/fair-ease/earth-critical-zone-galaxy/refs/heads/main/landcover.json"  
```

Comment:
why do you need to download it if you have it in this repo?

`$__tool_directory__/landcover.json`

Reply:
Because I love to complicate things 😂

Reply:
ok I preferred the reproducible way ;)